example/src/IosApp.tsx

```diff
@@ -0,0 +1,146 @@
+import * as React from 'react';
+import { sha256 } from 'react-native-sha256';
+import { StyleSheet, SafeAreaView, Text, ScrollView } from 'react-native';
+import {
+  generateHardwareKey,
+  getAttestation,
+  isAttestationServiceAvailable,
+  generateHardwareSignatureWithAssertion,
+} from '@pagopa/io-react-native-integrity';
+import ButtonWithLoader from './components/ButtonWithLoader';
+
+const challenge = 'challengeFromServer';
+
+export default function IosApp() {
+  const [hardwareKeyTag, setHardwareKeyTag] = React.useState<
+    string | undefined
+  >('');
+  const [attestation, setAttestation] = React.useState<string | undefined>('');
+  const [hsWithAssertion, setHsWithAssertion] = React.useState<
+    string | undefined
+  >('');
+  const [isServiceAvailable, setIsServiceAvailable] =
+    React.useState<boolean>(false);
+  const [debugLog, setDebugLog] = React.useState<string>('.. >');
+
+  React.useEffect(() => {
+    isAttestationServiceAvailable()
+      .then((result) => {
+        setIsServiceAvailable(result);
+      })
+      .catch((error) => {
+        setDebugLog(error);
```

Comment:
I think we cannot set this state by passing the raw error as it causes a runtime crash when an exception happens. You can try this running the example app on the emulator

Reply:
Yep. Accidentally It was fixed [here](https://github.com/pagopa/io-react-native-integrity/pull/6/commits/dfb1b09b48d8a8212c36e02112d5f868aac613bc#). I'll update this also.